spec/flows/certificated_passporting_spec.rb

```diff
@@ -78,6 +78,16 @@
     end
   end
 
+  it "asks for property details if relevant", :ee_banner, :shared_ownership do
```

Comment:
All my tests were around a non-passported flow so I have added this one to cover the changes in the passported flow around shared_ownership